src/Dock.Avalonia/HostWindowState.cs

```diff
@@ -76,7 +76,7 @@ private void Drop(Point point, DragAction dragAction, IVisual relativeTo)
 
         private void Leave()
         {
-            if (_targetDropControl is DockPanel)
+            if (_targetDropControl is Panel)
```

Comment:
We probably need to add custom Panel type here or something otherwise dock target is displayed on tabitem header